sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -107,6 +107,8 @@ class ServiceBusReceiver(
      specified Topic the client connects to.
     :keyword Optional[float] max_wait_time: The timeout in seconds between received messages after which the
      receiver will automatically stop receiving. The default value is None, meaning no timeout.
+     If there is a sessionful queue or topic and NEXT_AVAILABLE_SESSION is specified,
+     max_wait_time will replace the server-side operation timeout when waiting on connecting to a session.
```

Comment:
Maybe something like this ?

```suggestion
     If there is session enabled queue or topic and NEXT_AVAILABLE_SESSION is specified,
     max_wait_time will wait for that time to receive a message from any session before the operations times out.
```